QUARKUS-2994.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-2994 JDK 21 Support
+
+This document is a high level test plan for certification of Red Hat build of Quarkus on JDK 21.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Temurin JVM 21 and OpenJDK 21 need to be added as JDKs to Quarkus QE Jenkins
+* Temurin JVM 21 and OpenJDK 21 have to be added as JVMs to Polarion
+* Bare metal jobs in extended platform testing need to be extended to run with both Temurin 21 and OpenJDK 21.
+  * This affects JVM mode only.
+* Openshift JVm job needs to use OpenJDK 21 (together with existing 17) instead of OpenJDK 11
+  * We also should use s2i build image based on jdk 21 there
+
+### Impact on resources
+* Should not change (or be minimal), since for affected jobs we should drop testing on java 11, so consumed resource will stay the same.  
```

Comment:
While this is technically correct, I'd say this is really unhelpful for anybody who's not re-running our jobs over and over again. It would be nice to have information about lab machine utilization needed to ensure support of a new JDK.

Reply:
Added the calculations.

A a side note: readers of these plans are usually members of Quarkus QE, who are generally "re-running these jobs over and over again" during releases. Previously, tests plans focused on  **changes** in resource consumption, not the current level.